cmd/devnet/services/polygon/proofgenerator.go

```diff
@@ -11,6 +11,7 @@ import (
 	"sync"
 
 	"github.com/ledgerwatch/erigon/cl/merkle_tree"
+	bor_types "github.com/ledgerwatch/erigon/polygon/bor/types"
```

Comment:
please can we avoid using snake case for package names and import aliases? `bortypes` is how I would personally name an import alias like this and this is the style guide ive seen used widely in go os projects

Reply:
https://go.dev/blog/package-names

Reply:
https://google.github.io/styleguide/go/decisions.html#package-names

Reply:
This refers to package names, not package aliases.  These are different things.

The alias is just a local name for the package in the file to disambiguate it.

Reply:
it applies to aliases as well - otherwise I would not mention it
maybe we need a linter for this 

the point is - snake case is not followed in code, so it should not be followed as alias either because that means you end up having snake_case.Xyz() in code too

Reply:
here's an example warning - anyways, if you feel strongly about it then let's ignore this until we have some sort of team style consistency agreement (if that ever happens)

<img width="467" alt="Screenshot 2024-03-28 at 16 12 37" src="https://github.com/ledgerwatch/erigon/assets/94537774/19a8a518-4c00-482a-8aa3-85021fd065f3">